static/bidder-info/iqx.yaml

```diff
@@ -0,0 +1,19 @@
+endpoint: "http://rtb.iqzone.com/?pid={{.SourceId}}&host={{.Host}}&pbs=1"
```

Comment:
any specific reason to add `/` before query params `?`. Following should also work

`endpoint: "http://rtb.iqzone.com?pid={{.SourceId}}&host={{.Host}}&pbs=1"`